target/linux/ar71xx/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -58,6 +58,9 @@ case "$FIRMWARE" in
 		ath9k_eeprom_extract "art" 4096 2048
 		ath9k_patch_firmware_mac $(macaddr_add $(mtd_get_mac_binary art 0) +1)
 		;;
+	lima)
```

Comment:
This is redundant here, please use `ath79_register_wmac()` in mach-lima.c with correct offset.